homework_5/Methods/GuessWordGame.java

```diff
@@ -1,5 +1,35 @@
 package Methods;
 
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
 public class GuessWordGame {
-    
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
+        Random random = new Random();
+        String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado" , "broccoli", "carrot", "cherry",
```

Comment:
'words' array can be a constant on a class level. The reason to do so is that the values of this array are always the same according to the task description.